From: Aron Xu <aron@debian.org>
Date: Thu, 26 Jan 2012 01:55:34 +0800
Subject: [PATCH] use bash in header

---
 scripts/fcitx-fbterm-helper |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/scripts/fcitx-fbterm-helper b/scripts/fcitx-fbterm-helper
index b622e7e..b91a99e 100644
--- a/scripts/fcitx-fbterm-helper
+++ b/scripts/fcitx-fbterm-helper
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 function Usage()
 {
-- 
